.travis/dispatch.sh

```diff
@@ -52,6 +52,6 @@ else
     # Only run tests against the new configuration
     # TODO: don't hard-code this
     if [ "${SALT_NODE_ID}" = "servo-master1" ]; then
-        ./test.py sls.buildbot.master sls.nginx
+        ./test.py sls.buildbot.master sls.nginx sls.homu
```

Comment:
Let's keep these in alphabetical order, so insert `sls.homu` in between `sls.buildbot.master` and `sls.nginx`.